DQMServices/Core/interface/DQMEDHarvester.h

```diff
@@ -24,13 +26,14 @@ class DQMEDHarvester
   // implicit assignment operator
   // implicit destructor
   virtual void beginRun(edm::Run const&, edm::EventSetup const&) {};
-  virtual void analyze(edm::Event const&, edm::EventSetup const&) final {};
+  virtual void produce(edm::Event&, edm::EventSetup const&) override final {};
```

Comment:
when looking into more detail:
- you could drop now `beginLuminosityBlock` as this was only required by the `EDAnalyzer`
- `endLuminosityBlock` was called previously and is not called anymore because it is now an `EDProducer`